### PR TITLE
Edits from Ryan

### PR DESCRIPTION
**Pull request by RyanGutenkunst**
Edits throughout, but focused on DFE section.
Does require minor regeneration of those figures, presumably by @silastittes

### CODE REVIEW

File: analysis2_manuscript.tex [hunk 1]
```diff
@@ -696,8 +713,8 @@ \section*{Discussion}
     In this paper, we have presented an important new addition to the \stdpopsim{} library
     that allows for simulating genetic variation in the presence of selection.
     We have demonstrated the utility of this new API by showing how it can be used to benchmark
-    the performance of different methods for demographic inference, the inference of the distribution
-    of fitness effects of mutations, and the detection of selective sweeps.
+    the performance of different methods for demographic histtory inference, the inference of the distribution
```
Comment (andrewkern) [inline analysis2_manuscript.tex:716]:
```suggestion
    the performance of different methods for demographic history inference, the inference of the distribution
```

File: analysis2_manuscript.tex [hunk 2]
```diff
@@ -624,9 +640,10 @@ \section*{Detection of Selective Sweeps}
...
     thus the out-of-Africa bottleneck has does not mute the signal of a sweep as much as in the CEU sample. This is consistent with
-    what we know about the effects of demography on the power to detect sweeps \citep[e.g.,][]{simonsen1995properties}.
+    what is known about the effects of demography on the power to detect sweeps \citep[e.g.,][]{simonsen1995properties}.
     Second, the power to detect sweeps is slightly lower when we use a null model with purifying selection acting on exons (consistent with \cite{schrider2020background}) but the effect is not
-    as strong as one might expect. This is likely because the strength of selection acting on exons is relatively weak, and the
+    as strong as one might expect. 
+    \rngcomment{I'm confused by the terminology ``null model'' throughout this section. Are we always testing on simulated data that includes background selection, and ``null model'' refers to how the methods are tuned? Or are we comparing power on data simulated with and without background selection?} This is likely because the strength of selection acting on exons is relatively weak, and the
```
Comment (andrewkern) [inline analysis2_manuscript.tex:646]: the null model here refers to simulations with and without BGS to determine the threshold cutoff for CLR, etc.

File: analysis2_manuscript.tex [hunk 3]
```diff
@@ -642,16 +659,16 @@ \section*{Detection of Selective Sweeps}
     likely because that population size history more closely matches the population size history used to train \diploshic.
     \sweepfinder power remains more constant along the chromosome, while the $\pi$ cutoff method varies wildly. 
 
-    We can further explore the performance of the methods by plotting the receiver operating characteristic (ROC) curves for the methods.
+    We further explored the performance of the methods by plotting their receiver operating characteristic (ROC) curves.
     In Figure \ref{fig:roc-curves}, we show the ROC curves for the three methods for detecting sweeps.
     False positive rates were established using neutral simulations that contained no sweeps
-    but were conditioned on the same population history as the CEU and YRI samples.
-    These suggest that the $\pi$ cutoff is a suprisingly effective method for detecting sweeps,
+    but were conditioned on the same population history as the CEU and YRI samples. \rngcomment{Did these simulations contain background selection?}
```
Comment (andrewkern) [inline analysis2_manuscript.tex:665]: good question-- @mufernando are the ROC curves with BGS?

**Review by andrewkern — APPROVED** [review]
these changes look good to me. i've left one suggestion in the review and a couple of comments

**Comment by andrewkern** [comment]
@silastittes once this gets merged there are a couple figure updates to do. see @RyanGutenkunst's comments

**Comment by silastittes** [comment]
> @silastittes once this gets merged there are a couple figure updates to do. see @RyanGutenkunst's comments
Sounds good! I'll keep an eye out for when it's merged.

**Pull request merged**